spec/element_base_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::ElementBase do
+
+  let(:book) do
+    book_containing(html:
+      one_chapter_with_one_page_containing(
+        <<~HTML
+          <div data-type="example" class="class1" id="div1">
+            <p>This is a paragraph.</p>
+          </div>
+        HTML
+      ))
+  end
+
+  let(:example) { book.first!('div[data-type="example"]') }
+
+  let(:para) { book.first!('p') }
+
+  describe '#has_class?' do
+    it 'returns true if element has given class' do
+      expect(example.has_class?('class1')).to eq true
+    end
+
+    it 'returns false if element does not have given class' do
+      expect(example.has_class?('class2')).to eq false
+    end
+  end
+
+  describe '#id' do
+    it 'returns the element\'s ID' do
+      expect(example.id).to eq 'div1'
+    end
+  end
+
+  describe '#id=' do
+    it 'sets the element\'s ID' do
+      para = book.first!('p')
```

Comment:
remove (you have this available via the `let` above)

Reply:
ooooops